README.md

```diff
@@ -1,158 +1,278 @@
-# Java REST wrapper for Microsoft Azure Notification Hubs
+[ ![Download](https://api.bintray.com/packages/microsoftazuremobile/SDK/Notification-Hubs-Java-Backend-SDK/images/download.svg) ](https://bintray.com/microsoftazuremobile/SDK/Notification-Hubs-Java-Backend-SDK/_latestVersion)
 
+# Microsoft Azure Notification Hubs Java SDK
 
-Java Back-end SDK for [Microsoft Azure Notification Hubs](https://azure.microsoft.com/en-us/services/notification-hubs/) which wraps the [REST APIs](https://docs.microsoft.com/en-us/rest/api/notificationhubs/).
+This repository contains source code for the Java SDK for [Microsoft Azure Notification Hubs](https://azure.microsoft.com/en-us/services/notification-hubs/).
 
-Implements most of the Notification Hubs REST operations to perform Hub CRUDs, registration/installation management and send notification.
+### Note on FCM and GCM Support
 
-## What's New
+The Azure Notification Hubs SDK has support for Firebase Cloud Messaging (FCM) via the Legacy HTTP APIs which is compatible with GCM.  For registrations, use the `FirebaseRegistration` to create registrations, as the `GcmRegistration` class has been deprecated.  For more information, read the [Azure Notification Hubs and Google Firebase Cloud Messaging migration](https://docs.microsoft.com/en-us/azure/notification-hubs/notification-hubs-gcm-to-fcm)
```

Comment:
Could use the shortlink here

https://aka.ms/AA9dpaz

Reply:
Added